platform_versioned_docs/version-24.3/getting-started/studios.mdx

```diff
@@ -0,0 +1,452 @@
+    devtools::install_github('pinin4fjords/shinyngs', upgrade_dependencies = FALSE)
+    ```
+
+1. Download the RDS file from nf-core AWS megatests or your own nf-core/differentialabundance results (see [Shiny app](https://nf-co.re/differentialabundance/1.5.0/docs/output/#shiny-app) from the nf-core documentation for file details): 
+
+    ```r 
+    # For nf-core AWS megatests
+    download.file("https://nf-core-awsmegatests.s3-eu-west-1.amazonaws.com/differentialabundance/results-3dd360fed0dca1780db1bdf5dce85e5258fa2253/shinyngs_app/study/data.rds", 'data.rds')
+
+    # For your nf-core/differentialabundance results, replace the URL with your RDS file URL)
+    download.file("https://bucket.s3-region.amazonaws.com/differentialabundance/results/shinyngs_app/study-name/data.rds", 'data.rds')
+    ```
+
+1. Import libraries, read your RDS data, and launch the RShiny app:
+
+    ```r 
+    library(shinyngs)
+    library(markdown)
+    esel <- readRDS("data.rds")
+    app <- prepareApp("rnaseq", esel)
+    shiny::shinyApp(app$ui, app$server)
+    ```
+
+![Explore the RShiny app](./quickstart-demo/assets/rnaseq-diffab-rshiny-app-explore.gif)
+
+### Xpra: Visualize genetic variants with IGV
+
+Studios and Xpra remote desktop technology enables many interactive analysis and troubleshooting workflows. One such workflow is to perform genetic variant visualization using IGV desktop, a powerful open-source tool for the visual exploration of genomic data. This section demonstrates how to add public data from the 1000 Genomes project to your workspace, set up an Xpra environment with IGV desktop pre-installed, and explore a variant of interest. 
+
+#### Create an AWS Batch compute environment 
+
+Studios requires an AWS Batch compute environment. If you do not have an existing compute environment with at least 2 CPUs available, create one with the following attributes:
+
+- **Region**: To reduce costs, your compute environment should be in the same region as your data. To browse the 1000 Genomes public data optimally, select **us-east-1**.
+- **Provisioning model**: Use **On-demand** EC2 instances. 
+- Studios does not support AWS Fargate. Do not enable **Use Fargate for head job**. 
+- At least 2 available CPUs.
+
+#### Add data using Data Explorer
+
+Add the 1000 Genomes S3 bucket to your workspace using Data Explorer:
+
+1. From the **Data Explorer** tab, select **Add cloud bucket**. 
+1. Specify the bucket details:
+    - **Provider**: AWS
+    - **Bucket path**: `s3://1000genomes`
+    - A unique **Name** for the bucket, such as `1000G`.
+    - **Credentials**: Select **Public** from the dropdown menu.
+    - An optional bucket **Description**.
+1. Select **Add**.
+
+:::info 
+To use your own data for interactive analysis, see [Add a cloud bucket](./quickstart-demo/add-data.mdx#add-a-cloud-bucket) for instructions to add your own public or private cloud bucket. 
+:::
+
+#### Create an Xpra studio 
```

Comment:
Isn't the whole "killer feature" about XPRA the ability for multiple people to collaborate in the container within the same session? If I'm right, this seems like something you'd want to call out beyond the solo interaction steps currently documented.

Reply:
The killer feature of Xpra is also that it's a whole remote desktop with a GUI.

Reply:
Added a "Copy studio URL" section to each Studio type to highlight session collaboration.